Makefile

```
@@ -356,7 +356,8 @@ endef
 CERT_DIRS := receiver/sapmreceiver/testdata \
              receiver/signalfxreceiver/testdata \
              receiver/splunkhecreceiver/testdata \
-             receiver/mongodbatlasreceiver/testdata/alerts/certs
+             receiver/mongodbatlasreceiver/testdata/alerts/certs \
```

```suggestion
             receiver/mongodbatlasreceiver/testdata/alerts/cert \
```
Since this is being handled in another PR

---

That would be the fix in #12118? As currently written, the issue for that would still be open right?